### DISCUSSION TITLE
Input with action: button to the left instead of right

### DISCUSSION THREAD

**Discussion by smartm0use** [fomantic/Fomantic-UI#1290] [Elements (button, image, label, loader, ...)] [answered]
I'm looking for something like this:

![download](https://user-images.githubusercontent.com/5399265/73252329-de72a800-41ba-11ea-84b1-2698e05d0f3c.png)

With button on the left, instead of right.

According to [this](https://fomantic-ui.com/elements/input.html#action) it is not possible.
Do you have a way to reach that or do you plan to add more options to input with actions?

**Comment by dutrieux** [comment, marked as answer]
like this : https://jsfiddle.net/vntoa5hb/

**Comment by smartm0use** [comment]
Yes! Shouldn't it be added in the documentation as example?

**Comment by ColinFrick** [comment]
The second example shows an input with an action on the left:
![image](https://user-images.githubusercontent.com/5517677/73259056-3d3e1e80-41c7-11ea-9380-7341c4535be0.png)

**Comment by smartm0use** [comment]
You are right, my bad :(

Sorry if I go off-topic... do you know any way to make this button appear like "on/off".
I mean users click on it and turn it on, then if they click again they turn it off.
Maybe I can use borders or different colour. Do you have a better idea?

**Comment by ColinFrick** [comment]
I'd go with adding the "active" class to the button on click. There is also the toggle modifier to automatically change it to the color green when activating, or add another color class manually.
Additionally you could also change the icon inside the button:
https://jsfiddle.net/hn90szx5/

**Comment by smartm0use** [comment]
I see, thank you so much for your code!

What about adding space between button and input? Maybe you can solve [this](https://github.com/fomantic/Fomantic-UI/issues/1224) in a better way...

**Comment by smartm0use** [comment]
Something like this https://jsfiddle.net/sck6d3q8, but I think that `button` shoud not be included in the `div`.

**Comment by apple-idn** [comment]
```
<form class="ui form">
  <div class="field">
    <label>File</label>
    <div class="ui right action left icon input">
      <i class="folder open yellow icon"></i>
      <input type="text" placeholder="file" readonly>

      <button class="ui red button"><i class="erase icon"></i></button>
    </div>
  </div>
</form>
```
similar bug rounded input action left icon
<img width="352" alt="Jepretan Layar 2020-03-22 pukul 2 19 17 PM" src="https://user-images.githubusercontent.com/33564576/77243740-4c31d200-6c48-11ea-8b53-05a3c872b5fd.png">
